docs/running-node-ansible/vn.md

```diff
@@ -0,0 +1,167 @@
+# Running Validator Node using Ansible
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+
+## Configure DCL network parameters (local machine)
+#### 1. Set network chain id
+[`deployment/ansible/inventory/hosts.yml`]
+```yaml
+all:
+  vars:
+    chain_id: <chain-id>
+  ...
+```
+Every network must have a unique chain ID (e.g. `test-net`, `main-net` etc.)
+
+<details>
+<summary>Example for Testnet 2.0</summary>
+
+```yaml
+all:
+  vars:
+    chain_id: testnet-2.0
+  ...
+```
+</details>
+
+#### 2. Put `genesis.json` file under specific directory
+Get or download `genesis.json` file of a network your node will be joining and put it under the following path:
+```
+deployment/persistent_chains/<chain-id>/genesis.json
+```
+where `<chain-id>` is the chain id of a network spefied in the previous step
```

Comment:
Please specify the value for the existing Test Net. Please specify how to download it for the existing Test Net (here and in other docs).